client/package.json

```diff
@@ -8,18 +8,18 @@
   "dependencies": {
     "@bpmn-io/add-exporter": "^0.2.0",
     "@bpmn-io/align-to-origin": "^0.7.0",
-    "@bpmn-io/bpmn-properties-panel": "github:bpmn-io/bpmn-properties-panel#v0.5.2",
     "@bpmn-io/dmn-migrate": "^0.4.3",
     "@bpmn-io/extract-process-variables": "^0.4.3",
     "@bpmn-io/form-js": "^0.5.0",
+    "@bpmn-io/properties-panel": "^0.10.0",
     "@bpmn-io/replace-ids": "^0.2.0",
     "@ibm/plex": "^6.0.0",
     "@sentry/browser": "^6.3.6",
-    "bpmn-js": "^8.8.2",
-    "bpmn-js-properties-panel": "^0.46.0",
+    "bpmn-js": "^8.8.3",
+    "bpmn-js-properties-panel": "^1.0.0-alpha.0",
```

Comment:
Given we removed [this line](https://github.com/camunda/camunda-modeler/pull/2666/files#diff-f8ab767bafa564b8d50c1ae454c9f8fa0edaf15fcb206c5d2b3e2d4dee8daf81L11), do we anywhere use this library directly?

Or do we use this one for the plugin exposing?

Reply:
Apparently, yes 😄  https://github.com/camunda/camunda-modeler/pull/2632/files#diff-4087bdddf31860a72f6029df58cf6eb36c1dae0b29e122ae44a90b269553f3b9R29

Reply:
Purely to expose it to the plugins.